internal/scripts/scripts.go

```diff
@@ -169,9 +174,11 @@ func (c *ContentValue) Write(thread *starlark.Thread, fn *starlark.Builtin, args
 	}
 	fdata := []byte(data.GoString())
 
-	// No mode parameter for now as slices are supposed to list files
-	// explicitly instead.
-	err = os.WriteFile(fpath, fdata, 0644)
+	err = c.CreateFile(&fsutil.CreateOptions{
```

Comment:
Why has the comment been dropped?  Doesn't look like that bit has changed.

Reply:
Added it back.